tutorial_src/lea/config.yml

```diff
@@ -0,0 +1,9 @@
+title: Leaves
+
+memory_limit: 65536
+time_limit: 1000
+
+scores:
+	1: 10
```

Comment:
Tabulacje na spacje.

Reply:
Tzn niby zresolvowałeś ale u mnie dalej są taby z jakiegoś powodu.

Reply:
To powinny być taby, bo to yaml i on wymaga tabów